style.css

```diff
@@ -647,6 +652,6 @@ button:active {
 
   .navbard .mylogo {
     margin-left: 155px;
-    transform: rotate(15deg);
+    transform: rotate(20deg);
   }
```

Comment:
Please, this logo shows transform when hovering, this means that by default it should be normal😀